en/docs/setup/migrating-data-publishers.md

```diff
@@ -52,25 +52,37 @@ and migrating it to use the
 [AnalyticsSessionDataPublishHandler](https://github.com/wso2-extensions/identity-data-publisher-authentication/blob/master/components/org.wso2.carbon.identity.data.publisher.authentication.analytics.session/src/main/java/org/wso2/carbon/identity/data/publisher/authentication/analytics/session/AnalyticsSessionDataPublishHandler.java)
 instead.
 
-1.  Open the `           identity.xml          ` file found in the
-    `           <IS_HOME>/repository/conf/identity          ` folder and
-    remove the listener configuration corresponding to the data
-    publisher that you wish to migrate. For example, the following
-    listener configuration corresponding to the
-    [DASSessionDataPublisherImpl](https://github.com/wso2-support/identity-data-publisher-authentication/blob/support-5.1.7/components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/impl/DASSessionDataPublisherImpl.java)
-    has been removed in order to deprecate it.
+1.  Open the `identity.xml.j2` file found in the
+    `<IS_HOME>/repository/resources/conf/templates/repository/conf/identity` folder 
+    and add the template for removing the listener configuration corresponding to the data
+    publisher that you wish to migrate. For example, refer the following
+    listener configuration template corresponding to the
+    [DASSessionDataPublisherImpl](https://github.com/wso2-support/identity-data-publisher-authentication/blob/support-5.1.7/components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/impl/DASSessionDataPublisherImpl.java).
 
     ``` xml
-    <EventListener type="org.wso2.carbon.identity.core.handler.AbstractIdentityMessageHandler"
-    name="org.wso2.carbon.identity.data.publisher.application.authentication.impl.DASSessionDataPublisherImpl"
-    orderId="11" enable="false"/>
+    <EventListener id="abstract_identity_message_handler"
+        type="org.wso2.carbon.identity.core.handler.AbstractIdentityMessageHandler"
+        name="org.wso2.carbon.identity.data.publisher.application.authentication.impl.DASSessionDataPublisherImpl"
+        orderId="{{event.default_listener.abstract_identity_message_handler.priority}}"
+        enable="{{event.default_listener.abstract_identity_message_handler.enable}}"/>
     ```
+    
+    Navigate to `default.json` file found in the`<IS_HOME>/repository/resources/conf` folder and
+    add the values to the attributes specified in the `identity.xml.j2` file.
+  
+       ``` xml
+       "event.default_listener.abstract_identity_message_handler.priority": "11",
+       "event.default_listener.abstract_identity_message_handler.enable": false,
+       ```
+    !!! note "NOTE"
```

Comment:
No need to add the phrase "NOTE"

Reply:
If we didn't add `NOTE` it will just display in lowercase letters like `Note'. But if we add `NOTE` it will make sure all the letters are in the uppercase.

Reply:
The icon itself indicates that it's a note, so no need to explicitly mention it.

Reply:
fixed: 755e715a1bdd5ac9f01e6a73d74dc83761780cad